From 7c2d9d02e4f418413028fcae8f944ddd1b7ad32e Mon Sep 17 00:00:00 2001
From: Alvin Sun <alvinsunyixiao@gmail.com>
Date: Fri, 1 Jul 2022 13:30:21 -0700
Subject: [PATCH] customize debian rules files

---
 debian/rules.em | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/debian/rules.em b/debian/rules.em
index 284a071..edaa708 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -37,6 +37,13 @@ override_dh_auto_configure:
 	dh_auto_configure -- \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+		-DVRPN_BUILD_CLIENT_LIBRARY=ON \
+		-DVRPN_BUILD_JAVA=OFF \
+		-DVRPN_BUILD_PYTHON=OFF \
+		-DVRPN_BUILD_CLIENTS=OFF \
+		-DVRPN_BUILD_SERVERS=OFF \
+		-DVRPN_BUILD_SERVER_LIBRARY=OFF \
+		-DBUILD_TESTING=OFF \
 		$(BUILD_TESTING_ARG)
 
 override_dh_auto_build:
-- 
2.25.1

